models/programUsers.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    name: "programUsers",
+    schema: {
+      programId: {
+        type : "ObjectId",
+        required: true,
+        index: true
+      },
+      userId: {
+        type: String,
+        index: true
+      },
+      noOfResourcesStarted: {
+        type:Number,
+        index: true,
+        default: 0
+      },
+      userProfile: Object,
```

Comment:
@VISHNUDAS-tunerlabs this is also required, we can put a mandatory check on id inside this object maybe